test/controllers/healthcheck.controller.spec.ts

```diff
@@ -1,3 +1,5 @@
+jest.mock("ioredis");
```

Comment:
this was just to stop a warning message appearing when running the tests, message was about having to forcibly terminate tests due to redis connection

Reply:
Basically needed on all similar controller tests then?

Reply:
only if they use the middleware that connects to redis, you'll see a yellow warning message if there is a test that needs it mocked.